bench/GraphicsBench.java

```diff
@@ -0,0 +1,240 @@
+package bench;
```

Comment:
Nit: all the other benchmark classes in this directory are in the *benchmark* package (unless they're in a package from some part of the platform they need to access), so this one should be too.